package.json

```diff
@@ -247,6 +246,7 @@
     "xml2js": "^0.4.23"
   },
   "devDependencies": {
+    "@salesforce/sfdx-lwc-jest": "^1.0.1",
```

Comment:
codacy-eslint plugins need to all go to the `dependencies` sections since they are used in the tool at run-time during the analysis.